src/vs/workbench/services/authentication/browser/authenticationService.ts

```diff
@@ -428,23 +428,35 @@ export class AuthenticationService extends Disposable implements IAuthentication
 
 	async showGetSessionPrompt(providerId: string, accountName: string, extensionId: string, extensionName: string): Promise<boolean> {
 		const providerName = this.getLabel(providerId);
-		const { choice } = await this.dialogService.show(
-			Severity.Info,
-			nls.localize('confirmAuthenticationAccess', "The extension '{0}' wants to access the {1} account '{2}'.", extensionName, providerName, accountName),
-			[nls.localize('allow', "Allow"), nls.localize('deny', "Deny"), nls.localize('cancel', "Cancel")],
-			{
-				cancelId: 2
+		enum SessionPromptChoice {
+			Allow = 0,
+			Deny = 1,
+			Cancel = 2
+		}
+		const { result } = await this.dialogService.prompt<SessionPromptChoice>({
```

Comment:
@TylerLeonhardt fyi please check, this one is a bit more complex adoption of the new dialog api